forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/ApplicationAuditListener.java

```diff
@@ -76,12 +76,17 @@ protected Application prepareApplicationAudit(DelegateExecution execution) {
         String submitterName = String.valueOf(execution.getVariable(SUBMITTER_NAME));
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
         String submittedBy = null;
-        if (authentication instanceof JwtAuthenticationToken) {
-            submittedBy = ((JwtAuthenticationToken) authentication).getToken().getClaimAsString("preferred_username");
-            if(submittedBy.startsWith("service-account")){
-                submittedBy = "Anonymous-user";
+        if (authentication != null) {
+            if (authentication instanceof JwtAuthenticationToken) {
```

Comment:
as per the new java you can define variable in instanceof and use it
if (authentication instanceof JwtAuthenticationToken authToken){
submittedBy = authToken.getToken()...
}

Reply:
done